client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -43,6 +44,7 @@ import { Component, createRef } from 'react';
 import { connect } from 'react-redux';
 import QueryActivePromotions from 'calypso/components/data/query-active-promotions';
 import FoldableCard from 'calypso/components/foldable-card';
+import SelectDropdown from 'calypso/components/select-dropdown';
```

Comment:
This is creating another dependency on a non-packaged Calypso component. It will need to be refactored: either move to `@automattic/components`, localise/move under plans-features-2023-grid, or find some other component from Core or the components package.

Reply:
> This is creating another dependency on a non-packaged Calypso component. It will need to be refactored: either move to @automattic/components

Thanks for the heads up 👍 

I'll get started on that in a separate PR that we can incorporate into the changes in this PR

Reply:
cool. but maybe do check if an alternative dropdown component exists in either `@automattic/components` or Wordpress Core and see if that should suffice already?

Reply:
for Core/Wordpress components, there's Storybook online to quickly review choices https://wordpress.github.io/gutenberg 

I can see some [CustomSelectControl](https://wordpress.github.io/gutenberg/?path=/story/components-customselectcontrol--default) that might have same look & feel

Reply:
> but maybe do check if an alternative dropdown component exists in either @automattic/components or Wordpress Core

Yeah I didn't see anything obvious in @automattic/components when I skimmed through it earlier, but the `Core/Wordpress` component looks promising

Reply:
I went with the @wordpress/components package [9caaedc](https://github.com/Automattic/wp-calypso/pull/79041/commits/9caaedc955273d49b07a4a26bc1f8b0ee53223d2)

Reply:
Sorry in case I steered you in a wrong direction here (with the Core component) 😬 

Both of these (local Calypso and Core) are heavily subject to custom styling. Ideally, we want to have consistent look across the app, so need to keep that in mind. It might be the ideal option to migrate the existing Calypso component under `@automattic/components`, especially if designers find that the suggested look & feel (vs restyling the Core one to match this case alone). Either way, I think something _will_ need to migrate under `@automattic/components` eventually.

Reply:
> Sorry in case I steered you in a wrong direction here (with the Core component) 😬

No need to apologize here -- I agreed with you and figured @wordpress/components was the right way to go as well.

I started the migration in the following PRs:

- https://github.com/Automattic/wp-calypso/pull/79255#top
- https://github.com/Automattic/wp-calypso/pull/79257#top
- https://github.com/Automattic/wp-calypso/pull/79256#top

We can switch out `@wordpress/components` to `@automattic/components` in a future PR when the above PRs are merged